lib/stat_tracker.rb

```diff
@@ -41,5 +41,48 @@ def lowest_total_score
     lowest_score
   end
 
+  def percentage_home_wins
+    games = 0
+    home_wins = 0
+    @games_data.each do |game|
+      games += 1
+      if game[:home_goals].to_i > game[:away_goals].to_i
+        home_wins += 1
+      end
+    end
+
+    if games != 0
+      (home_wins.to_f / games * 100.0).round(1)
+    end
+  end
+
+  def percentage_visitor_wins
+    games = 0
```

Comment:
We might use a different variable name when refactoring. Concerned about confusion with just simply "games"